sql/core/src/main/scala/org/apache/spark/sql/expressions/Window.scala

```diff
@@ -159,6 +159,17 @@ object Window {
   }
 
   /**
+   * @param start boundary start, inclusive. The frame is unbounded if this evals to
+   *              the minimum long value (`Window.unboundedPreceding`).
+   * @param end boundary end, inclusive. The frame is unbounded if this evals to the
+   *            maximum long value (`Window.unboundedFollowing`).
+   * @since 2.3.0
+   */
+  def rowsBetween(start: Column, end: Column): WindowSpec = {
```

Comment:
hmmm, how useful is this API? `rowsBetween` only accept integer and max/min long, and it's ok if we only have `rowsBetween(start: Long, end: Long)`